webapp/src/main/webapp/WEB-INF/jsp/login.jsp

```diff
@@ -20,7 +20,7 @@
 
 <p>You asked for a protected resource. Please log in:</p>
 
-<form action="/j_security_check" method="POST">
+<form action="/doLogin" method="POST">

```

Comment:
Bra! Også enklere å forstå, for de som ikke har mye erfaring med Acegi/Spring Security...